### PR TITLE
Fix OES_texture_float to GL_OES_texture_float.

### LINKED ISSUE #9234: [Report] Hardware Tessellation test on android failed.
Just simple report.
I tried auto-built v1.3-321-gce2feb2 but it's failed with Hardware Tessellation.
I hope to fix it but I don't know how to fix and debug with android.

Tested bezier in pspgl on ZenFone 5(Android 4.4.2)

screenshot_2017-01-25-05-13-38

### PR DESCRIPTION
**Pull request by xebra**
I hope to fix #9234.

I think OES_texture_float is only for WebGL extensions, correct extension is GL_OES_texture_float.
Sorry, I was misunderstanding.

### CODE REVIEW

**Comment by xebra** [comment]
If this fix is correct, this may need to fix as same reason.
`gl_extensions.OES_texture_npot = strstr(extString, "OES_texture_npot") != 0;`
Never mind if this extension is working correctly.

**Comment by hrydgard** [comment]
Yeah, GL documentation is generally very inconsistent about prefixed. Some sources even mention functions like glClear as just Clear which is very confusing. This should be correct and you are right about npot.

File: GPU/GLES/GPU_GLES.cpp [hunk 1]
```diff
@@ -600,7 +600,7 @@ void GPU_GLES::CheckGPUFeatures() {
 	if (maxVertexTextureImageUnits >= 3) // At least 3 for hardware tessellation
 		features |= GPU_SUPPORTS_VERTEX_TEXTURE_FETCH;
 
-	if (gl_extensions.ARB_texture_float || gl_extensions.OES_texture_float)
+	if (gl_extensions.ARB_texture_float || gl_extensions.OES_texture_float || gl_extensions.OES_texture_half_float)
```
Comment (unknownbrackets) [inline GPU/GLES/GPU_GLES.cpp]: According to https://www.khronos.org/registry/OpenGL/extensions/OES/OES_texture_float.txt, OES_texture_half_float only means half-float is supported, not float.  Presumably no device would support half float and not single float, but there's no reason for this check IMHO.

-[Unknown]

Comment (xebra) [inline GPU/GLES/GPU_GLES.cpp]: Sorry, I didn't know about half float is needed or not. We should remove this?

Comment (hrydgard) [inline GPU/GLES/GPU_GLES.cpp]: I don't think you're using half-float anywhere, so can just ditch the check.

**Pull request merged**